### PR TITLE
translate "install:pkginstallers" and "install:githubreleases"

### PR DESCRIPTION
**Pull request by 65hz**
# What did I do?
I translated the remaining English text in the page with Portuguese text.

---

shoutouts to my mum btw she helped when my grammar blew ass <3

### CODE REVIEW

**Review by SMillerDev — DISMISSED** [review]

File: _data/locales/pt.yml [hunk 1]
```diff
@@ -19,8 +19,8 @@ pt:
       install: Instale o Homebrew
       paste: Cole este código numa linha de comandos do seu Terminal macOS ou Linux.
       what: O script explica o que irá fazer e faz uma pausa antes de ser executado. Leia sobre outras <a href="https://docs.brew.sh/Installation">opções de instalação</a>.
-      pkginstaller: If you're on macOS, try our new <code>.pkg</code> installer.
-      githubrelease: Download it from <a href="https://github.com/Homebrew/brew/releases/latest">Homebrew's latest GitHub release</a>.
+      pkginstaller: Se tu és em macOS, tenta o novo instalador do .pkg<code>.pkg</code>.
```
Comment (EricFromCanada) [inline _data/locales/pt.yml:22]:
```suggestion
      pkginstaller: Se tu és em macOS, tenta o novo instalador do <code>.pkg</code>.
```

File: _data/locales/pt.yml [hunk 2]
```diff
@@ -19,8 +19,8 @@ pt:
...
+      githubrelease: Baixe das<a href="https://github.com/Homebrew/brew/releases/latest">últimas lançamentas da GitHub da Homebrew.</a>.
```
Comment (EricFromCanada) [inline _data/locales/pt.yml:23]:
```suggestion
      githubrelease: Baixe das <a href="https://github.com/Homebrew/brew/releases/latest">últimas lançamentas da GitHub da Homebrew</a>.
```

**Review by EricFromCanada — DISMISSED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks for the PR @65hz! Once these tiny issues are resolved should get this merged ❤️

File: _data/locales/pt.yml (hunk 1 above)
Comment (65hz) [reply _data/locales/pt.yml:22]: pkg is already written down in the `<code> </code>` block

**Review by 65hz — COMMENTED** [review]
Added some changes to areas where I messed up

**Review by 65hz — COMMENTED** [review]
committed the changes

**Review by 65hz — COMMENTED** [review]
realized why it wasn't working (I didn't press viewed, new to GitHub sorry)
(I pressed review idk what happened)

**Review by MikeMcQuaid — APPROVED** [review]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @65hz!

**Pull request merged**